examples/England-PractitionerRole-NHS-Example.xml

```diff
@@ -0,0 +1,25 @@
+<PractitionerRole xmlns="http://hl7.org/fhir">
```

Comment:
GeneralPractitioner is a better alternative to NHS

Reply:
Pushed change. Renamed to be more appropriate